Add CloseConfirm v1.0.0 and RememberMe v1.0.0 and ScalableMaterialOrbs v1.0.0 and Deprecate VRLogixRotateFix

AlexW-578 opened the pull request:
- [x] Follow the [schema](https://www.neosmodloader.com/schema). An automated check will validate this PR against the schema.
- [x] Indent using four spaces. An automated check will validate this PR indents properly.
- [x] Your NeosMod.Version should match the version in the manifest.
- [x] You should not monopack libraries into your mod DLL.
- [x] Do not remove old versions.
- [x] Do not remove old mods. Instead, use the `deprecated` [flag](https://www.neosmodloader.com/manifest-flags).
- [x] Indicate platform incompatibilities using the appropriate [flag](https://www.neosmodloader.com/manifest-flags) (for example `broken:linux-native`)
- [x] Follow the [privacy and security guidelines](https://www.neosmodloader.com/mod-guidelines#privacy-and-security)
- [x] In order to reduce merge conflicts avoid adding mods at the very end of the file.

AlexW-578 committed:
Add CloseConfirm and RememberMe

EIA485 commented on manifest.json:
```diff
@@ -6692,6 +6692,52 @@
+        "co.uk.AlexW-578.CloseConfirm": {
+            "name": "CloseConfirm",
+            "description": "This stops the close button on the window as well as alt-f4 from closing the game but rather opening up the dash to the exit screen.",
+            "category": "Misc",
```
probably fits best in `Keybinds & Gestures`

AlexW-578 replied:
Sure, I wasn't 100% sure which category that they would fall into.
I'll change it to be that then

AlexW-578 committed:
Change category for CloseConfirm

AlexW-578 committed:
Add ScalableMaterialOrbs v1.0.0 and Deprecate VRLogixRotateFix

Deprecating VRLogixRotateFix in favour of it being combined with LogixUtils v1.7.0
